BookStore.UnitTests/Application/Services/BookServiceTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AutoFixture;
+using AutoMapper;
+using BookStore.Application.DTOs;
+using BookStore.Application.Services.Books;
+using BookStore.Domain.Models;
+using BookStore.Domain.Repositories;
+using FluentAssertions;
+using Moq;
+
+namespace BookStore.UnitTests.Application.Services
+{
+    public class BookServiceTests
+    {
+        private readonly Fixture _fixture;
+
+        public BookServiceTests()
+        {
+            _fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task GetAllBooksAsync_ShouldReturnListOfBooks()
+        {
+            // Arrange
+            var expectedBooks = _fixture.CreateMany<Book>(2);
+            var bookRepositoryMock = new Mock<IBookRepository>();
+            bookRepositoryMock.Setup(repo => repo.GetAllBooksAsync()).ReturnsAsync(expectedBooks);
+
+            var mapperMock = new Mock<IMapper>();
+            mapperMock.Setup(mapper => mapper.Map<IEnumerable<BookDto>>(It.IsAny<IEnumerable<Book>>()))
```

Comment:
I have already mentioned multiple times not to mock mapper. That makes any test irrelevant